.scala-steward.conf

```
@@ -0,0 +1,5 @@
+updates.pin = [
+  { groupId = "co.fs2", artifactId="fs2-core", version="2." },
+  { groupId = "io.circe", artifactId="circe-fs2", version="0.13." },
```

I think we need to pin all of Circe
```suggestion
  { groupId = "io.circe", version="0.13." },
```